packages/storybook/stories/Stateful Client/OverviewStateful.stories.mdx

```diff
@@ -81,7 +81,7 @@ The device manager allows developers to access the local video stream to render
 const cameras = await deviceManager.getCameras();
 const camera = cameras[0];
 const localCameraStream = { source: camera, mediaStreamType: 'Video' } as LocalVideoStreamState;
-const view = statefulClient.createView(localCameraStream);
```

Comment:
So this story was... simply wrong before this PR.

Reply:
Yep lol

Reply:
Really we need to be able to include sub snippets of snippets so the few lines we want to show have been compile-checked

Reply:
Or perhaps a snippet view that can only show a few lines and then has an "expand" button that shows the whole snippet, imports etc. included